docs/application/tizen-studio/native-tools/code-coverage.md

```diff
@@ -1,44 +1,75 @@
+   All the unit tests present in the Unit Test project are executed and coverage is determined based on the code covered by the unit tests:
 
    ![Profiling the application](./media/code_coverage_profile.png)
 
-2. After the application is launched, run the application scenario you want to test.
+2. After the Unit Test project completes its execution, Tizen Studio displays the code coverage information in the editor. The **Unit Test Result** and **Code Coverage** views provide a summary of the unit test results and code coverage percentage for each file.
 
-   ![Application running](./media/code_coverage_run.png)
+   The code covered during the test execution has a green highlight and the uncovered code has a red highlight as shown in the following figures:
 
-3. To get the coverage report, press the **Stop/Cancel Operation** button in the **Progress** view of Tizen Studio.
+   **Figure: Covered code highlighted with green and uncovered code with red**
 
-   ![Stop/Cancel Operation button](./media/code_coverage_end.png)
+   ![Code Coverage result](./media/code_coverage_result.png)
+   
+   **Figure: Detailed view of Code Coverage**
 
-4. Tizen Studio shows the code coverage information in the editor after the application has closed. A **Code Coverage** view also opens to give a summary of the code coverage percentage for each file.
+   ![Code Coverage result](./media/code_coverage_result_11.png)
 
-   The code covered during the scenario has a green highlight and the uncovered code has a red highlight.
+    Each row of the **Code Coverage** view can be expanded to view the function level coverage information as shown in the following figure:
 
-   ![Code Coverage result](./media/code_coverage_result.png)
-
-   Each row of the **Code Coverage** view can be expanded to show the function level coverage information.
+    **Figure: Code Coverage result on a function level**
 
    ![Code Coverage result on a function level](./media/code_coverage_result2.png)
 
+   The Unit Test results are shown in **Native Unit Test Result** view as shown in the following figure:
+    
+    **Figure: Unit Test Result**
+
+   ![Unit Test Result](./media/code_coverage_unittest_11.png)
+
+## Generate HTML Code Coverage report
+
+To Enable HTML code coverage report generation: 
+
+1. Enable the checkbox option **Enable HTML Report generation when code coverage is run** from menu **Window > Preferences > Tizen Studio > Code Coverage** as shown in the following figure:
+
+    ![Code Coverage HTML Menu](./media/code_coverage_html_menu_11.png)
+    
+    > [!NOTE]
+    > To annotate the source files when opened from **Project Explorer**, select the checkbox **Enable annotation when source files are opened** in the preferences dialog box.
+
+2. Run the Code coverage again by following the steps mentioned in previous section. After the execution completes, the HTML reports are generated under **Unit_test_project_path/Coverage_Report/Html_Report**:
```

Comment:
since this is varying how about angular brackets usage: Unit_test_project_path

2. Run the Code coverage again by following the steps mentioned in the previous section. After the execution completes, the HTML reports are generated under **\<Unit_test_project_path\>/Coverage_Report/Html_Report**, where **\<Unit_test_project_path\>** is the path where the Unit Test project is created.

Reply:
@vivek-ellur Can you please resolve the comment?